source/crud/tests/write/findOneAndUpdate-arrayFilters.yml

```diff
@@ -11,16 +11,16 @@ tests:
             arguments:
                 filter: {}
                 update:
-                    $set: {y.$[i].b: 2}
+                    $set: {"y.$[i].b": 2}
                 arrayFilters:
                     - {i.b: 4}
 
         outcome:
             result:
                 _id: 1
                 y:
-                    - {b:3}
-                    - {b:1}
+                    - {b: 3}
```

Comment:
The Ruby yaml parser also is unable to parse documents from strings without a space after the colon.